src/guiFormSpecMenu.cpp

```diff
@@ -308,7 +334,7 @@ void GUIFormSpecMenu::parseList(parserData* data,std::string element)
 		else
 			loc.deSerialize(location);
 
-		v2s32 pos = padding + AbsoluteRect.UpperLeftCorner;
+		v2s32 pos = padding + AbsoluteRect.UpperLeftCorner + pos_offset * spacing.X;
```

Comment:
Don't multiply with the .X scalar of the spacing, but instead do the element wise schur product, notation is `pos_offset * spacing`. See also [irrlicht source](http://irrlicht.sourceforge.net/docu/vector2d_8h_source.html#l00057)